BCA/totalchecking.py

```diff
@@ -0,0 +1,38 @@
+import os
+import glob
+import  pandas as pd
+
+def checkTheSum75():
+    all_files_name=glob.glob("*.json")
+    currentPath=os.getcwd()
+    noFoYears=[]
+    check75=0
+    #loop for opening file 
+    for l in all_files_name:
+        print("checking for Year : ",l)
+        sampleFile=pd.read_json(currentPath+"\\"+l)
+        
+        #loop for counting no of year in each file
+        for i in range(len(sampleFile['topics'])):
+            for j in range(len(sampleFile['topics'][i]['weights'])):
```

Comment:
when using indexing like this, try storing them in a meaningful variable like
```python
weights = sampleFile['topics'][i]['weights']
```
which makes it easy to reference them later in the code